### PR TITLE
feat: Updating paths for Gains and CollectionMetadata

### PR DESCRIPTION
**Pull request by manasaV3**
Relates to https://github.com/chanzuckerberg/cryoet-data-portal/issues/898

### Frames
- Update the location of .mdoc to `/<dataset-id>/<run-name>/Frames`
### Gains
- Update gain files to be stored in `/<dataset-id>/<run-name>/Gain`
- Update the naming of the gain file to be `<name-in-source>.<ext>`

### CODE REVIEW

**Review by Bento007 — APPROVED** [review]

File: ingestion_tools/scripts/tests/fixtures/dataset1.yaml [hunk 1]
```diff
@@ -23,6 +23,11 @@ annotations:
     file_format: csv
     glob_string: particle_lists/{run_name}_fas.csv
     shape: Point
+collection_metadata:
```
Comment (jgadling) [inline ingestion_tools/scripts/tests/fixtures/dataset1.yaml:26]: is the code for collection_metadata support not part of this PR?

**Review by jgadling — APPROVED** [review]
Seems like there's an extra file in thIs PR?

File: ingestion_tools/scripts/tests/fixtures/dataset1.yaml (hunk 1 above)
Comment (manasaV3) [reply ingestion_tools/scripts/tests/fixtures/dataset1.yaml:26]: Good catch! the collection_metadata test file got missed in being included.
